name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/BaaderBankPDFExtractor.java

```diff
@@ -225,6 +225,11 @@ private void addDividendeTransaction()
                     t.setAmount(asAmount(v.get("amount")));
                     t.setDateTime(asDate(v.get("date")));
                 })
+                
+                // Ex-Dividenden-Tag
```

Comment:
// Ex-Dividenden-Tag
not the same for regEx
`^Ex-Tag: (?<exDate>\\d+.\\d+.\\d{4})$`

Reply:
changed